### PR TITLE
Delete deprecated methods

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Change list

Delete deprecated methods
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/io/appium/java_client/pagefactory/DefaultElementByBuilder.java [hunk 1]
```diff
@@ -166,27 +166,17 @@ protected By buildMobileNativeBy() {
         Optional<HowToUseLocators> howToUseLocatorsOptional = ofNullable(howToUseLocators);
 
         By result = null;
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/pagefactory/DefaultElementByBuilder.java]: is this variable still used?

**Comment by mykola-mokhnach** [comment]
It also makes sense to check the failing job on Azure CI: https://dev.azure.com/srinivasansekar/java-client/_build/results?buildId=74

File: src/main/java/io/appium/java_client/pagefactory/DefaultElementByBuilder.java (hunk 1 above)
Comment (SrinivasanTarget) [reply src/main/java/io/appium/java_client/pagefactory/DefaultElementByBuilder.java]: Nope, it's not used, have deleted it now.

**Comment by mykola-mokhnach** [comment]
Checkstyle is still not very happy

**Pull request merged**